### PR TITLE
Builder landing page

### LINKED ISSUE #404: Adding a description of what the lab is when non-logged in users navigate there
Currently all you get is 'You need to be signed in to use the lab' if you navigate to the lab page when not logged in. It might be worth a better description of what the lab is especially for new researchers that might be checking out the Zooniverse home page but don't yet have an account. 

screen shot 2015-05-25 at 11 07 55 am

### PR DESCRIPTION
**Pull request by srallen**
- Added landing page component based on @mrniaboc's design
  - Logged out users or users that have not built a project will see the landing page when they use "Build a project" link from the header.
- "Build a project" nav item is visible to logged out users
- Fixed #404

### CODE REVIEW

**Comment by srallen** [comment]
Sorry just noticed an issue... I'll work on it.

**Comment by srallen** [comment]
OK this can be reviewed now. Sorry about that.

**Comment by brian-c** [comment]
No worries, I'll check it out a little later today. Thanks.

File: app/pages/lab/landing-page.cjsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    document.documentElement.classList.remove 'on-secondary-page'
+    document.documentElement.classList.remove 'on-landing-page'
+
+  render: ->
+    <div className="landing-page">
+      <ZooniverseLogoType />
+      <h3 className="landing-title"><Translate content="landing.title" /></h3>
+      <p className="landing-tagline"><Translate content="landing.content" /></p>
+      {if @props.user?
+        <Link to="build" className="call-to-action standard-button landing-button" onClick={unless @props.user? then @showLoginDialog}><Translate content="landing.buttons.getStarted" /></Link>
```
Comment (brian-c) [inline app/pages/lab/landing-page.cjsx]: The `#/build` routes should actually be removed, everything is off `#/lab` now. I should have done that a while ago, sorry. I guess "Get started" should create a new project?

**Comment by brian-c** [comment]
Looks like some background sizing business got weird:

screen shot 2015-06-19 at 2 17 23 pm

And possibly unrelated to these changes, but I noticed the disabled sign-in button turned white on almost-white:

screen shot 2015-06-19 at 2 24 54 pm

**Comment by brian-c** [comment]
Fixed my grumbles since Sarah's away. Reassigning...

**Comment by chrissnyder** [comment]
:+1:

**Pull request merged**